File: app/src/main/java/vgame/ir/data/local/entity/TimeSpentEntity.java
package vgame.ir.data.local.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.SerializedName;

@Entity(tableName = "time_spent")
public class TimeSpentEntity {

    @PrimaryKey(autoGenerate = true)
    @SerializedName("id")
    private int id;

    @SerializedName("lesson_id")
    private int lesson_id;

    @SerializedName("time")
    private long time;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getLesson_id() {
        return lesson_id;
    }

    public void setLesson_id(int lesson_id) {
        this.lesson_id = lesson_id;
    }

    public long getTime() {
        return time;
    }

    public void setTime(long time) {
        this.time = time;
    }
}

File: app/src/main/java/vgame/ir/data/local/entity/TimeSpentEntity.kt
package vgame.ir.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

import com.google.gson.annotations.SerializedName

@Entity(tableName = "time_spent")
class TimeSpentEntity {

    @PrimaryKey(autoGenerate = true)
    @SerializedName("id")
    var id: Int = 0

    @SerializedName("lesson_id")
    var lesson_id: Int = 0

    @SerializedName("time")
    var time: Long = 0
}